### PR TITLE
Caasp4 more providers

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: playbooks/kvm-deploy_caasp4.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        tasks_from: setup_terraform_workspace_libvirt
+      vars:
+        skuba_ci_terraform_worker_count: "{{ deploy_on_kvm_caasp_workers }}"
+        skuba_ci_terraform_workspace: "{{ socok8s_workspace }}/skuba-terraform/"
+
+    - import_role:
+        name: caasp4
+        tasks_from: setup_nodes
+      vars:
+        _import_terraform_resources: false
```
Comment (Itxaka) [inline playbooks/kvm-deploy_caasp4.yml]: This seems like it should be a normal var. The `_` seems to indicate that is temporary, butr no other declared var from playbooks (other that commmand outputs) is using this, so its a bit confusing to track it down.

Comment (toabctl) [reply playbooks/kvm-deploy_caasp4.yml]: Removed the `_`. Marking as solved conversation for now.

**Comment by toabctl** [comment]
@Itxaka could you review again please?

File: playbooks/kvm-delete_caasp4.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- hosts: localhost
+  gather_facts: yes
+  any_errors_fatal: true
```
Comment (evrardjp) [inline playbooks/kvm-delete_caasp4.yml]: Do we need this? If true, can you explain why in a comment?

Comment (toabctl) [reply playbooks/kvm-delete_caasp4.yml]: I don't think we need it. Copy&paste artifact.

**Pull request merged**